packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -122,6 +128,33 @@ export interface VercelServerlessConfig {
 
 	/** The maximum duration (in seconds) that Serverless Functions can run before timing out. See the [Vercel documentation](https://vercel.com/docs/functions/serverless-functions/runtimes#maxduration) for the default and maximum limit for your account plan. */
 	maxDuration?: number;
+
+	/** Whether to cache on-demand rendered pages in the same way as static files. */
+	isr?: boolean | VercelISRConfig;
+}
+
+interface VercelISRConfig {
+	/**
+	 * A random string that you create.
+	 * Its presence in the `__prerender_bypass` cookie will result in fresh responses being served, bypassing the cache.
+	 * 
+	 * By default, none.
+	 */
+	bypassToken?: string;
```

Comment:
I assume this value should always remain secret? Do we want to clarify that in the JSDoc comment?